crmsh/cibconfig.py

```diff
@@ -816,26 +816,49 @@ def postprocess_cli(node, oldnode=None, id_hint=None):
     resolve_references(node)
     if oldnode is not None:
         remove_id_used_attributes(oldnode)
+    if complete_advised:
+        complete_advised_meta(node)
     return node, obj_type, obj_id
 
 
+def complete_advised_meta(node):
+    """
+    Complete advised meta attributes
+    """
+    if node.tag != "clone":
+        return
+    primitive_list = node.xpath('primitive')
+    if not primitive_list:
+        return
+    if is_promotable(node):
```

Comment:
IMO as long as "meta promotable=" is explicitly set by the user, no matter what value it is. we should probably leave it and return, since that's what is apparently needed by the user.

Reply:
Already changed
Thanks!